client/orm/model_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package orm
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type ModelWithComments struct {
+	Id       int    `orm:"description(user id)"`
+	UserName string `orm:"size(30);unique;description(user name)"`
+	Email    string `orm:"size(100)"`
+	Password string `orm:"size(100)"`
+}
+
+func TestGetDbCreateSQLWithComment(t *testing.T) {
```

Comment:
nice, but you can improve the unit tests.
Firstly, use table driven to write the unit tests, for example
```go
testCases := []struct{
    name string
    wantSQL string
    wantErr error
}
for _, tc := range testCases {
   t.Run(tc.name, func(t) {
    // ...
})
}
```
Secondly, do not use `modelCache`. The unit tests should be independent, but if you use the `modelCahe`, other tests may be affected by your test. So you can create a new `_modelCache`.

Thirdly, you need to think about the scenarios, for example, no comment, empty comment...

Finally, write the expected SQL directly, it's not necessary to use symbol "+" to construct the expected SQL

Reply:
Thanks for your advice! I'll try to fix these issues.